packages/core/addon/components/navi-visualizations/table.js

```diff
@@ -173,6 +173,35 @@ class Table extends Component {
     }, []);
   }
 
+  /**
+   * Adds meta into rows for serverside data rollups.
+   *
+   * @param {Array<Row>} tableData - Rows of data to render
+   * @param {RequestV1} request - request in order
+   * @returns {Array<Row>} - tableData with added meta to denote rollup rows
+   */
+  _mapRollups(tableData, request) {
+    if (request?.rollup?.columns?.length <= 0 && !request?.rollup?.grandTotal) {
+      return tableData;
+    }
+    const fullMask = parseInt(new Array((request.dimensions?.length ?? 0) + 1).fill('1').join(''), 2);
```

Comment:
This logic works on the mirage side, but with duplicate dimension/metrics this breaks the UI side, can we add a test for that?

Reply:
Okay fixed it. I also added a duplicate dimension to unit test and made sure it breaks the old way.